Wikipedia/Code/ArticlePeekPreviewViewController.swift

```diff
@@ -177,6 +177,7 @@ class ArticlePeekPreviewViewController: UIViewController {
             // Open in new tab
             let openInNewTabAction = UIAction(title: CommonStrings.articleTabsOpenInNewTab, image: WMFSFSymbolIcon.for(symbol: .tabsIcon), handler: { [weak self] _ in
                 guard let self = self else { return }
+                articleTabsDataController.didTapOpenNewTab()
```

Comment:
Maybe add this same call below within the `openInNewBackgroundTabAction`, so the same tracking happens when tapping "open in background tab".